finders/metadata/esi-funds.json

```diff
@@ -9,6 +9,11 @@
   "filter": {
     "document_type": "european_structural_investment_fund"
   },
-  "default_order": "-closing_date",
```

Comment:
Yes, thought it was one of the lines of the old format but now realising it's not. The funds still looked ordered as I hadn't republished them. 
Now investigating and in the meantime setting this PR to [DO NOT MERGE]. Thanks for spotting it!